backend/src/test/java/com/htecgroup/skynest/service/impl/UserServiceImplTest.java

```diff
@@ -203,4 +191,29 @@ void resetPassword() {
 
     Assertions.assertEquals(expectedResponse, userService.resetPassword(anyString(), anyString()));
   }
+
+  @Test
+  void deleteUser_UserWithIdNotFound() {
+    when(userRepository.existsById(any())).thenReturn(false);
+
+    Assertions.assertThrows(UserException.class, () -> userService.deleteUser(null));
```

Comment:
some mock value is always better then **_null_**, because null can have some different meaning

Reply:
This is unit test, so no database is included here :) You are mocking here what will happen if **_existsById_** returns false so any value can be used. If you need UUID , you can always create one with `UUID.randomUUID()`

Reply:
You are completely right